src/.pre-commit-config.yaml.jinja

```
@@ -85,6 +85,8 @@ exclude: |
   readme/.*\.(rst|md)$|
   # Ignore build and dist directories in addons
   /build/|/dist/|
+  # Ignore test files in addons
+  /tests/fixtures/*|
```

Thanks for the explanations.

I think "fixtures" is very confusing here, as it has no special meaning, and otherwise it reminds to pytest fixtures, which have no relationship with what you want.

What about this?

```suggestion
  /tests/vendor/*|
```

"Vendor" is a common word for external code or files that are included in an alien repo.

---

Fine for me. @tuantrantg can you update?

---

I would recommend `/tests/resources` as a more generic name for files which the tests depend on. This directory needn't necessarily contain 3rd-party stuff. Failing that, `/tests/vendor` is also fine.

---

Or even `samples` better than `vendor` one IMO.